{
  "timestamp": 1526125206282,
  "status": 500,
  "error": "Internal Server Error",
  "exception": "com.alibaba.dubbo.rpc.RpcException",
  "message": "Failed to invoke the method getUserName in the service com.alkaid.dtmp.sso.api.UserService. Tried 3 times of the providers [192.168.127.1:20882] (1/1) from the registry 192.168.43.150:2181 on the consumer 192.168.127.1 using the dubbo version 2.8.4. Last error is: Failed to invoke remote method: getUserName, provider: dubbo://192.168.127.1:20882/com.alkaid.dtmp.sso.api.UserService?accepts=1000&anyhost=true&application=dtmp-sso-web&check=false&dubbo=2.8.4&generic=false&interface=com.alkaid.dtmp.sso.api.UserService&methods=getUserName&module=dtmp-sso-web&optimizer=com.alkaid.dtmp.common.serial.SerializationOptimizerImpl&organization=dubbox&owner=alkaid&pid=3348&revision=1.0.0&serialization=kryo&side=consumer&timestamp=1526110627985&version=1.0.0, cause: Failed to send message Request [id=241, version=2.0.0, twoway=true, event=false, broken=false, data=RpcInvocation [methodName=getUserName, parameterTypes=[class com.alkaid.dtmp.sso.dao.model.User], arguments=[com.alkaid.dtmp.sso.dao.model.User@51ac3460], attachments={path=com.alkaid.dtmp.sso.api.UserService, interface=com.alkaid.dtmp.sso.api.UserService, version=1.0.0}]] to /192.168.127.1:20882, cause: com/alibaba/dubbo/common/serialize/support/kryo/CompatibleKryo",
  "path": "/login"
}